### PR TITLE
ab: add input box for duration time

### LINKED ISSUE #446: Define variables: start time needs its own units
The start time variable needs its own units and can't share a units dropdown with duration.

### PR DESCRIPTION
**Pull request by annagav**
Fix #446

### CODE REVIEW

File: instructor/common.py [hunk 1]
```diff
@@ -449,7 +450,7 @@ def assignments_edit_treatments(request):
         'g/L',
         'nM',
         u'\u03BCM', #uM
-        'mm',
+        'mM',
```
Comment (pdpinch) [inline instructor/common.py:538]: is this the abbreviation for millimeters??

Comment (pdpinch) [inline instructor/common.py:538]: Nevermind -- it's a concentration not a distance. It looks like it's millimolar.

**Comment by pdpinch** [comment]
I'm pretty sure this is working. I can't test it on a published course because I'm waiting on #524 and #528 so I can't confirm if the changes to compiler.py are working. 

I can test again after the others are merged and this is rebased.

**Comment by annagav** [comment]
Rebased!

**Comment by pdpinch** [comment]
This is working both in preview and in student view. 

One thing I just noticed though: it would be nice if there were a space between the value & the units:

screen shot 2015-12-30 at 2 17 04 pm

You can either fix that, or merge this and open an issue for the spacing.

Otherwise :+1:

**Pull request merged**